### PR TITLE
[fabric] Generation of configtx.yaml with usage of custom templates

### PR DESCRIPTION
**Pull request by jopuneet**
**Generation of configtx.yaml with usage of custom templates**
- Addition of new tasks for copy and checking the configtx.yaml
- Renaming configtx templates with default postfix and associated code
- Addition of configtx custom folder path

Co-authored-by: S m, Aruna <aruna.mohan@walmart.com>
Co-authored-by: Shubhkarman Walia <shubhkarman.walia@Walmart.com>
Co-authored-by: Karthik Samaganam <karthik.samaganam@walmart.com>

Signed-off-by: S m, Aruna <aruna.mohan@walmart.com>
Signed-off-by: Puneet Joshi <puneet.joshi@walmart.com>

### CODE REVIEW

File: platforms/hyperledger-fabric/configuration/samples/network-fabric-add-organization.yaml [hunk 1]
```diff
@@ -128,6 +132,23 @@ network:
       ca_data:
         url: ca.supplychain-net.org1ambassador.blockchaincloudpoc.com:8443
         certificate: file/server.crt        # This has not been implemented in 0.2.0.0
+
+      # Generating User Certificates with custom attributes using Fabric CA in BAF for Peer Organizations
```
Comment (arsulegai) [inline platforms/hyperledger-fabric/configuration/samples/network-fabric-add-organization.yaml]: Maybe this is not part of the current commit?

File: platforms/hyperledger-fabric/configuration/samples/network-fabricv2-raft.yaml [hunk 2]
```diff
@@ -228,6 +232,22 @@ network:
         certificate: file/server.crt
 
       cloud_provider: aws   # Options: aws, azure, gcp, digitalocean, minikube
+      # Generating User Certificates with custom attributes using Fabric CA in BAF for Peer Organizations
+      users:
```
Comment (arsulegai) [inline platforms/hyperledger-fabric/configuration/samples/network-fabricv2-raft.yaml]: Maybe this is not part of the current commit?

File: platforms/hyperledger-fabric/configuration/samples/network-fabricv2.yaml [hunk 3]
```diff
@@ -203,6 +207,22 @@ network:
         certificate: file/server.crt
 
       cloud_provider: aws   # Options: aws, azure, gcp, digitalocean, minikube
+      # Generating User Certificates with custom attributes using Fabric CA in BAF for Peer Organizations
+      users:
```
Comment (arsulegai) [inline platforms/hyperledger-fabric/configuration/samples/network-fabricv2.yaml]: Maybe this is not part of the current commit?

**Review by arsulegai — APPROVED** [review]
Looks good, other than adding users section in this commit. Looks like non-feature related change to the network yaml files.

File: platforms/hyperledger-fabric/configuration/samples/network-fabric-add-organization.yaml (hunk 1 above)
Comment (jopuneet) [reply platforms/hyperledger-fabric/configuration/samples/network-fabric-add-organization.yaml]: Sur. Removed these changes

**Review by arsulegai — APPROVED** [review]

**Pull request merged**